community/server/src/docs/ops/security.asciidoc

```diff
@@ -51,15 +51,13 @@ The Neo4j server includes built in support for SSL encrypted communication over
 The first time the server starts, it automatically generates a self-signed SSL certificate and a private key.
 Because the certificate is self signed, it is not safe to rely on for production use, instead, you should provide your own key and certificate for the server to use.
 
-To provide your own key and certificate, replace the generated key and certificate, or change _neo4j.conf_ to set the location of your certificate and key:
+To override the send-signed key and certificate, replace _neo4j.key_ and _neo4j.cert_ in the <<file-locations, _certificates_>> directory. The location of this directory can be configured by setting _dbms.directories.certificates_ in <<file-locations, _neo4j.conf_>>.
```

Comment:
I think you need to make it clearer here that the files must have the names `neo4j.key` and `neo4j.cert`.